stanzas/text/metadata.json

```
@@ -19,72 +19,80 @@
       "stanza:required": true
     },
     {
-      "stanza:key": "mode",
+      "stanza:key": "data-type",
       "stanza:type": "single-choice",
-      "stanza:choice": ["text", "markdown"],
-      "stanza:example": "text",
-      "stanza:description": "Mode",
+      "stanza:choice": ["json", "tsv", "csv", "sparql-results-json"],
+      "stanza:example": "json",
+      "stanza:description": "Data type",
       "stanza:required": true
     },
+    {
+      "stanza:key": "custom_css_url",
+      "stanza:example": "",
+      "stanza:description": "Stylesheet (css file) URL to override current style",
+      "stanza:required": false
+    },
     {
       "stanza:key": "highlight-css-url",
```

（念の為、ここにコメントします）
先ほど永野さんの指摘のように、`highlight_css_url`に変更をお願いいたします。
```suggestion
      "stanza:key": "highlight-css-url",
```